### PR TITLE
renamed and added comparisons to SUSE Manager

### PR DESCRIPTION
**Pull request by jhhudso**
## Description

I renamed the SMT and RMT comparison document and added a third tool comparison to SUSE Manager.

## Change Type

*Please select the correct option.*

- [x] **Documentation Update** (a change which only updates documentation)

## Other Notes

I thought it would be helpful for SUSE Linux Enterprise Server users to see how RMT versus SUSE Manager compares in-case someone is trying to decide between using RMT vs SUSE Manager to mirror SUSE maintenance updates and manage their SUSE Linux Enterprise Server systems.

### CODE REVIEW

**Comment by Sergeykot** [comment]
@mcalmer we would appreciate a SUMA expert to review listed info, I was wondering if you could help us with that

File: docs/smt_rmt_suma.md [hunk 1]
```diff
@@ -1,51 +1,64 @@
...
+|Registration of CentOS 8 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of CentOS 7 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of CentOS 6 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of Oracle Linux 8 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of Oracle Linux 7 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of Oracle Linux 6 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of openSUSE Leap 15 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of Ubuntu 20.04 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of Ubuntu 18.04 systems|:x:|:x:|:heavy_check_mark:|
+|Registration of Ubuntu 16.04 systems|:x:|:x:|:heavy_check_mark:|
```
Comment (paususe) [inline docs/smt_rmt_suma.md:36]: I'd simplify this part of the table to "non-SLES products (RHEL, Ubuntu, etc)" or move openSUSE above and then say "non-SUSE products (RHEL, Ubuntu, etc)". The reason is if you include the full list of SUSE Manager clients, the table is huge: RHEL, Oracle Linux, CentOS, AlmaLinux, Rocky Linux, Ubuntu, Debian, Amazon Linux, Alibaba Cloud Linux, and we will keep adding :-)

File: docs/smt_rmt_suma.md [hunk 2]
```diff
@@ -1,51 +1,64 @@
...
+|Support for migrating SLE 12 systems to 15|:warning:<sup>[1](#partial-migration)</sup>|:heavy_check_mark:|:heavy_check_mark:|
+|Support for migrating SLE 15 SPx systems to 15 SPx+1|:warning:<sup>[1](#partial-migration)</sup>|:heavy_check_mark:|:heavy_check_mark:|
+|Staging repositories|:heavy_check_mark:|:x:<sup>[2](#staging)</sup>|:heavy_check_mark:|
+|Air gap sync/mirroring for secure environments|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|NTLM Proxy support|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|Custom repositories|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|YaST installation wizard|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|YaST management wizard|:heavy_check_mark:|:x:|:x:|
```
Comment (paususe) [inline docs/smt_rmt_suma.md:44]:
```suggestion
|management wizard|:heavy_check_mark: (YaST)|:x:|:heavy_check_mark: (SUMA WebUI)|
```

File: docs/smt_rmt_suma.md [hunk 3]
```diff
@@ -1,51 +1,64 @@
...
+|Client management|:heavy_check_mark:|:x:|:heavy_check_mark:|
+|Red Hat 7 and earlier support ([Expanded Support](https://www.suse.com/products/expandedsupport/))|:heavy_check_mark:|:x:|:heavy_check_mark:|
+|Red Hat 8 support ([Expanded Support](https://www.suse.com/products/expandedsupport/))|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
```
Comment (paususe) [inline docs/smt_rmt_suma.md:47]: Maybe move this to the above part, together with the clients?

File: docs/smt_rmt_suma.md [hunk 4]
```diff
@@ -1,51 +1,64 @@
...
+|Files deduplication|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
```
Comment (paususe) [inline docs/smt_rmt_suma.md:48]:
```suggestion
|File deduplication|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
```

File: docs/smt_rmt_suma.md [hunk 5]
```diff
@@ -1,51 +1,64 @@
...
+|Transfer registration data to SCC|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
```
Comment (paususe) [inline docs/smt_rmt_suma.md:49]:
```suggestion
|Transfer registration data to SCC|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark: (optional)|
```

File: docs/smt_rmt_suma.md [hunk 6]
```diff
@@ -1,51 +1,64 @@
...
+|Reporting|:heavy_check_mark:|:x:|:heavy_check_mark:|
+|Custom TLS certificates for web-server|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|Clean up data from repositories that are not used any longer|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|Bash completion|:x:|:heavy_check_mark:|:x:|
+|Available on [openSUSE Leap 15](https://github.com/SUSE/rmt/blob/master/docs/installation.md#installation-on-opensuse-leap-15)|:x:|:heavy_check_mark:<sup>[3](#self-support)</sup>|:x:|
```
Comment (paususe) [inline docs/smt_rmt_suma.md:54]:
```suggestion
|Available on [openSUSE Leap 15](https://github.com/SUSE/rmt/blob/master/docs/installation.md#installation-on-opensuse-leap-15)|:x:|:heavy_check_mark:<sup>[3](#self-support)</sup>|:heavy_check_mark:<sup>[3](Uyuni, #self-support)|
```

File: docs/smt_rmt_suma.md [hunk 7]
```diff
@@ -1,51 +1,64 @@
...
+|Option to [run as container](https://github.com/SUSE/rmt/blob/master/README.md#development-setup---docker-compose)|:x:|:heavy_check_mark:<sup>[3](#self-support)</sup>|:x:|
+|Easy development setup + [contribution guide](https://github.com/SUSE/rmt/blob/master/docs/CONTRIBUTING.md)|:x:|:heavy_check_mark:|:x:|
```
Comment (paususe) [inline docs/smt_rmt_suma.md:56]:
```suggestion
|Easy development setup |:x:|:heavy_check_mark: [link](https://github.com/SUSE/rmt/blob/master/docs/CONTRIBUTING.md)|:heavy_check_mark: [link](https://github.com/uyuni-project/uyuni/wiki/Uyuni-development-in-no-time)|
```

File: docs/smt_rmt_suma.md [hunk 8]
```diff
@@ -1,51 +1,64 @@
...
+|100% test [coverage](https://coveralls.io/github/SUSE/rmt?branch=master)|:x:|:heavy_check_mark:|:x:|
+|[Plugin functionality](https://github.com/SUSE/rmt/blob/master/docs/PLUGINS.md)|:x:|:heavy_check_mark:|:x:|
```
Comment (paususe) [inline docs/smt_rmt_suma.md:58]: SUSE Manager has plugins for some things, e. g. virtual host managers and cluster managers

**Review by paususe — COMMENTED** [review]
The comparison could be expanded if you want to include more SUSE Manager functionality: DEB repository mirroring, configuration management, automation, Salt, Ansible, provisioning, virtualization management, etc

**Comment by Sergeykot** [comment]
thanks for your input @paususe

**Comment by ngetahun** [comment]
@jhhudso Can you please update the PR?

**Comment by felixsch** [comment]
Thanks your contributions! To get the changes added I created a new up to date pull request: https://github.com/SUSE/rmt/pull/745

Closing in favor of: https://github.com/SUSE/rmt/pull/745

**Pull request closed without merging**